### ISSUE TITLE
ConsumerOptions.AfterSync: provide default NOOP impl?

### ISSUE THREAD

**Issue by mxmCherry** [bsm/feedx#9]
https://godoc.org/github.com/bsm/feedx#ConsumerOptions - looks like AfterSync must ALWAYS be set.

Though I think that it's quite fine not to specify it at all, and work cast consumer's `Data()` for every usage.

Should there be a default no-op implementation for it (in `ConsumerOptions.norm()` I guess)?

**Comment by dim** [comment]
Yep, will address

**Comment by mxmCherry** [comment]
@dim I'd vote for some logging:

```go
// myAfterSync logs sync status.
func myAfterSync(updated bool, err error) {
	if err != nil {
		log.Errorw("sync", zap.Error(err), zap.String("feed", "URL?"), zap.Bool("updated", updated))
	} else {
		log.Infow("sync", zap.String("feed", "URL?"), zap.Bool("updated", updated))
	}
}
```

**Comment by dim** [comment]
@mxmCherry I thought about it but decided not to do anything. Each project should have its own way of dealing with this, you can define a global SyncHandler in common

**Comment by dim** [comment]
Addressed in #11
